src/main/java/org/mule/service/http/impl/util/HttpUtils.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.service.http.impl.util;
+
+import static org.mule.service.http.impl.service.server.grizzly.MuleSslFilter.SSL_SESSION_ATTRIBUTE_KEY;
+import org.mule.runtime.core.api.util.StringUtils;
+import org.mule.runtime.http.api.domain.request.ClientConnection;
+import org.mule.runtime.http.api.server.ServerAddress;
+import org.mule.service.http.impl.service.server.DefaultServerAddress;
+import org.mule.service.http.impl.service.server.grizzly.DefaultClientConnection;
+
+import java.net.InetSocketAddress;
+
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.SSLSession;
+
+import org.glassfish.grizzly.Connection;
+import org.glassfish.grizzly.ssl.SSLUtils;
+
+public final class HttpUtils {
```

Comment:
Are you actually using this? All this methods seems to have been taken from other places, except I don't see any equivalent deletions.

Reply:
A lot of methods were not used and I just delete them. A few are used in the EE part. In my experience, utility methods of low added value are better kept as available as possible. Otherwise one day you'll find yourself repeating it here for a path that cannot depend on EE